rename_in_folders

```diff
@@ -0,0 +1,17 @@
+import pathlib
+import re
+
+rom_root = pathlib.Path('/my/rom/root/folder/')
+
+folders_to_rename = ['gb', 'gbc', 'gba', 'megadrive', 'msx', 'n64',
```

Comment:
I like the idea with this, though not every device contains the same folders. Maybe the user could input the folders they'd like to rename the files in, add those to a list, and _then_ have the code iterate through that particular list.